db/db_impl.cc

```diff
@@ -3085,6 +3095,10 @@ Status DBImpl::GetLatestSequenceForKey(SuperVersion* sv, const Slice& key,
     return Status::OK();
   }
 
+  // We could do a sv->imm->GetEarliestSequenceNumber(/*include_history*/ true)
+  // check here to skip the history if possible. But now the caller already did
```

Comment:
s/But now the caller already did that/But currently the caller already does that/